index.html

```diff
@@ -3,30 +3,27 @@
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-
- 
   <title>Github Jobs API</title>
-
- <a href="http://href="https://fonts.googleapis.com/css?family=Dosis:400,700" rel="stylesheet""></a>
- 
-  <script src="scripts.js"></script>
-  
+  <a href="http://href="https://fonts.googleapis.com/css?family=Dosis:400,700" rel="stylesheet"></a>
   <link rel="stylesheet" type="text/css" href="style.css">
-  
 </head>
 
 <body>
    
   <!-- Search bar -->
-  <div class="wrap">
-   <div class="search">
+  <div>
+    <div class="search">
+      <br><br>
+      
       <input id ="userInput" type="text" class="searchTerm" placeholder="Keywords, Title, Career"/>
       <button type="submit" class="searchButton">
-        <i class="fa fa-search"></i>
+        Search
      </button>
-      </div>
-</div><br>
+      
+     <br>
+    </div>
+  </div><br>
 
       <!-- DIV FOR CARD DISPLAY -->
-     <div id="root"></div>
-  
+  <div id="root"></div>
+  <script src="scripts.js"></script>
```

Comment:
moved this line to bottom of HTML file so line 6 (const app = document.getElementById('root');) can be loaded directly without error.